components/breadcrumb/stories/breadcrumb.mdx

```diff
@@ -0,0 +1,56 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as BreadcrumbStories from './breadcrumb.stories';
+
+<Meta of={BreadcrumbStories} title="Docs" />
+
+<Title of={BreadcrumbStories} />
+<Description of={BreadcrumbStories} />
+
+### Standard
+
+<Canvas of={BreadcrumbStories.Default} />
+
+### Dragged
+
+<Canvas of={BreadcrumbStories.Dragged} />
+
+### Nested
+
+<Canvas of={BreadcrumbStories.Nested} />
+
+### Nested (root visible)
+
+<Canvas of={BreadcrumbStories.NestedRootVisible} />
+
+### Multiline
+
+<Canvas of={BreadcrumbStories.Multiline} />
+
+### Multiline (dragged)
+
+<Canvas of={BreadcrumbStories.MultilineDragged} />
+
+### Multiline Nested
+
+<Canvas of={BreadcrumbStories.NestedMultiline} />
```

Comment:
For each story, I used whatever heading was on the docs site. But those names don't always align with previously written stories (for example, the heading on the docs site is "Standard," but the story name is "Default." My view for this first batch of work was to just match what's on the docs site, even if it was a little inconsistent, but I'm open to changing things to make them consistent.